src/EditorFeatures/Core/SymbolSearch/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -41,6 +41,8 @@ internal partial class SymbolSearchUpdateEngine
         public const string HostId = "RoslynNuGetSearch";
         private const string MicrosoftAssemblyReferencesName = "MicrosoftAssemblyReferences";
 
+        private const int AddReferenceDatabaseTextFileFormatVersion = 1;
```

Comment:
`AddReferenceDatabase.TextFileFormatVersion` is removed and replaced by an instance method [`AddReferenceDatabase.GetFileFormatVersion`](https://github.com/microsoft/elfie-arriba/blob/master/Elfie/Elfie/Model/AddReferenceDatabase.cs#L35)

Reply:
> AddReferenceDatabase.TextFileFormatVersion is removed 

Instead of copying the constant... why not call .GetFileFormatVersion?

Reply:
Because we don't have a database object to call upon, now it's an instance method, instead of a constant.

Reply:
the point though is that we don't want roslyn to particularly know or care about thsi.  i.e. if the elfie format changes (which is something we have discussed), we can simply update our library reference and *not* touch any roslyn code. 

can this not be exposed as a static-readonly from teh elfie lib?  it does'nt have to be an instance method, but it shouldn't be a value we're hardcoding into roslyn itself.

Reply:
I understand. But our goal here is simply making it possible for us to properly target netstandard/netcoreapp so Roslyn can run on coreclr, and this does just that w/o any change to current behavior. If and when the database we download does change (which means someone is actually working on that side), we will then handle that as required. Until then, I think this is sufficient.

Reply:
In terms of define our own constant, I think it is consistent with current design and how we use the constant defined in Elfie today, i.e. we construct this link knowing the exact version the file will be at that location, it shouldn't be depends on some value we get from Elfie library.

Reply:
FYI @jinujoseph I will merge this as is, given that 
1. it preserves current behavior
2. there's no plan from nuget team to change the server side